app/services/moves/import_people.rb

```diff
@@ -2,10 +2,10 @@
 
 module Moves
   class ImportPeople
-    attr_accessor :items
+    attr_accessor :people_nomis_prison_numbers
```

Comment:
this name doesn't roll of the tongue easily, how about `nomis_prisoner_numbers` ?

Reply:
:D yeah prison_numbers is enough